src/Global/molecules/TopBar.tsx

```diff
@@ -130,6 +132,9 @@ const NotificationsContainer = styled.div`
     margin-right: -17px;
 `;
 
+type IDrawerKeys = 'user' | 'notifications' | 'custom' | '';
```

Comment:
What is the empty string being used for here? Seems like it's better to allow for a null or undefined value instead?

Reply:
makes a lot of sense, I'll update.